bundle/pom.xml

```diff
@@ -281,16 +282,25 @@
         <dependency>
             <groupId>org.apache.poi</groupId>
             <artifactId>poi-ooxml</artifactId>
-            <version>3.6</version>
+            <version>3.16</version>
```

Comment:
is this compatible with AEM 6.2? As far as I can tell, the POI version shipped with 6.2 was 3.11.

Reply:
AEM 6.3 ships with 3.15 and for my project Adobe provided an update with 3.16.  I can downgrade it to 3.11, it shouldn't be a problem. 
Compatibility shouldn't be a problem. I'm telling it with my Apache POI committer hat on.